src/data/multiset/antidiagonal.lean

```diff
@@ -67,6 +67,19 @@ quotient.induction_on s $ λ l, begin
   {congr; simp}, {simp}
 end
 
+theorem antidiagonal_eq_map_powerset [decidable_eq α] (s : multiset α) :
+   s.antidiagonal = s.powerset.map (λ t, (s - t, t)) :=
+ begin
+   induction s using multiset.induction_on with a s hs,
+   { simp only [antidiagonal_zero, powerset_zero, zero_tsub, map_singleton] },
+   { simp_rw [antidiagonal_cons, powerset_cons, map_add, hs, map_map, function.comp, prod.map_mk,
+       id.def, sub_cons, erase_cons_head],
+     rw add_comm,
+     congr' 1,
+     refine multiset.map_congr rfl (λ x hx, _),
+     rw cons_sub_of_le _ (mem_powerset.mp hx) }
+ end
+
```

Comment:
This entire proof has one extra space of indent (as does another proof in this PR); can you fix the indents?

Reply:
Oops, sorry about that. I don't know how that happened...